backend/src/seeds/seed.ReportEntryToCourseReports.ts

```diff
@@ -0,0 +1,55 @@
+import { ReportEntryToCourseReport } from '../report-entry-to-course-report/entities/report-entry-to-course-report.entity';
```

Comment:
This file doesn't follow `camelCase` convention.

Reply:
@bibixx what do you mean? You mean the dashes in the entity file?

Reply:
The file should be named `seed.reportEntryToCourseReports.ts`